src/System.Xml.ReaderWriter/src/System/Xml/Core/XmlEncodedRawTextWriterAsync.cs

```diff
@@ -1127,37 +1127,97 @@ protected unsafe int RawTextNoFlush(string text, int index, int count)
             }
         }
 
-        protected Task RawTextAsync(string text)
+        protected Task RawTextAsync(string text) // special-case the one string overload, as it's so common
```

Comment:
nit: please put the comment above or inside the method as reformatters and style analysis tools often complain about